### PR TITLE
Feature/grw 731 tooltip

### PR DESCRIPTION
**Pull request by guilhermespopolin**
## What?

- Updates `Tooltip` by using a dependency that takes care of best positioning  a tooltip on the screen
- Adds that tooltip to `CancellationOptions` as part of CPS

**Ticket(s): [GRW-731](https://hedvig.atlassian.net/browse/GRW-731)**

## Screenshots / recordings 

https://user-images.githubusercontent.com/19200662/156191033-ed11762f-abd3-426c-89e5-e91af0473134.mov

### CODE REVIEW

File: src/client/components/Tooltip/TooltipIcon.stories.tsx [hunk 1]
```diff
@@ -1,25 +1,28 @@
...
   },
 }
 
-const Template: Story<TooltipProps> = (args) => <Tooltip {...args} />
+const Template: Story<TooltipIconProps> = (args) => <TooltipIcon {...args} />
 
 export const Large = Template.bind({})
 Large.args = {
   body: 'For more information, see the full terms and conditions above',
+  placement: 'auto',
```
Comment (guilhermespopolin) [inline src/client/components/Tooltip/TooltipIcon.stories.tsx:21]: If some _Storybook_ wizard knows how to make this accepts a list with possible values of `placement` props, please let me know 😬 

Screenshot 2022-03-01 at 15 54 50

Comment (styrelius) [reply src/client/components/Tooltip/TooltipIcon.stories.tsx:21]: You add an `argTypes` object to the default export - have a look at `Footer.stories.tsx` for an example :)

File: src/client/components/Tooltip/TooltipIcon.tsx [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+const TooltipIconWrapper = styled.div`
+  /* remove extra space under child SVG: https://stackoverflow.com/a/51161925 */
+  font-size: 0;
+  line-height: 0;
+`
+
+export type TooltipIconProps = Pick<TooltipProps, 'body' | 'placement'>
+
+export const TooltipIcon = (props: TooltipIconProps) => {
+  const [isVisible, setVisible] = useState(false)
```
Comment (styrelius) [inline src/client/components/Tooltip/TooltipIcon.tsx:19]: I understand this is not written by you now but 
```suggestion
  const [isVisible, setIsVisible] = useState(false)
```

**Review by styrelius — APPROVED** [review]
Nice 👍

File: src/client/components/Tooltip/TooltipIcon.stories.tsx (hunk 1 above)
Comment (guilhermespopolin) [reply src/client/components/Tooltip/TooltipIcon.stories.tsx:21]: Noice. Thank you :)

**Review by styrelius — COMMENTED** [review]
This might be good enough but in Figma the cancellation options info icon are aligned with the top of the label text, and in this implementation it's aligned with the bottom of the label :)

**Figma:**
image


**This implementation:**
image

**Pull request merged**